project.clj

```diff
@@ -6,20 +6,20 @@
             :url "https://opensource.org/licenses/MIT"}
 
   :dependencies [[org.clojure/clojure "1.8.0"]
-                 [org.clojure/clojurescript "1.7.228"]
-                 [org.clojure/core.async "0.1.346.0-17112a-alpha"]
-                 [org.zeromq/jeromq "0.3.4"]
-                 [cheshire "5.4.0"]
-                 [org.omcljs/om "0.8.8"]
+                 [org.clojure/clojurescript "1.9.229"]
+                 [org.clojure/core.async "0.2.391"]
+                 [org.zeromq/jeromq "0.3.6"]
+                 [cheshire "5.6.3"]
+                 [org.omcljs/om "0.9.0"]
```

Comment:
What version of React is used by this version? There were some fairly big changes in React 15 that might effect us.